README.md

```diff
@@ -0,0 +1,117 @@
+# specfile
+
+Python library for parsing and manipulating RPM spec files. Main focus is on modifying existing spec files, any change should result in a minimal diff.
```

Comment:
I'd add a sentence or two about motivation. Something like `Previously, [rebase-helper](https://github.com/rebase-helper/rebase-helper/) provided an API for the modifications which was also used by [packit](https://github.com/packit/packit). The goal is to make the interface more general and convenient to use by these projects but also by other python projects which need to interact with RPM spec files.`

Reply:
Ok, I'd just add it as another paragraph.